### PR TITLE
Relayer: IExecutionQueue for messages and deployments

### PR DESCRIPTION
**Pull request by rzadp**
`IExecutionQueue` that can hold both deployments and messages, which are executed by `IExecutor` implementations

### CODE REVIEW

File: universal-login-relayer/lib/integration/ethereum/MessageExecutor.ts [hunk 1]
```diff
@@ -2,12 +2,18 @@ import {Wallet, providers} from 'ethers';
 import {SignedMessage} from '@universal-login/commons';
 import IMessageValidator from '../../core/services/validators/IMessageValidator';
 import {messageToTransaction} from '../../core/utils/messages/serialisation';
+import {QueueItem} from '../../core/models/QueueItem';
+import {IExecutor} from '../../core/services/execution/IExecutor';
 
-export class MessageExecutor {
+export class MessageExecutor implements IExecutor<SignedMessage> {
```
Comment (JustynaBroniszewska) [inline universal-login-relayer/lib/integration/ethereum/MessageExecutor.ts]: Let's name it `Executor` or `ExecutionService`

Comment (rzadp) [reply universal-login-relayer/lib/integration/ethereum/MessageExecutor.ts]: Are you sure? My idea was to have `MessageExecutor implements IExecutor<SignedMessage>`, and then introduce `DeploymentExecutor implements IExecutor<Deployment>`

Comment (JustynaBroniszewska) [reply universal-login-relayer/lib/integration/ethereum/MessageExecutor.ts]: Aaaa, ok

File: universal-login-relayer/lib/core/services/messages/IExecutionQueue.ts [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import {SignedMessage} from '@universal-login/commons';
+import {QueueItem} from '../../models/QueueItem';
+import Deployment from '../../models/Deployment';
+
+export interface IExecutionQueue {
+  addMessage (item: SignedMessage) : Promise<string>;
+  addDeployment (item: Deployment) : Promise<string>;
```
Comment (marekkirejczyk) [inline universal-login-relayer/lib/core/services/messages/IExecutionQueue.ts]: I guess we could unify this method in the future, and make both SignedMessage and Deployment IExecution or sth similar

File: universal-login-relayer/lib/integration/sql/services/QueueSQLStore.ts [hunk 3]
```diff
@@ -21,6 +22,17 @@ export default class QueueSQLStore implements IMessageQueue {
     return messageHash;
   }
 
+  async addDeployment(deployment: Deployment) {
```
Comment (marekkirejczyk) [inline universal-login-relayer/lib/integration/sql/services/QueueSQLStore.ts]: extract `add({hash, type})`

**Pull request merged**